### PR TITLE
DOC : chart deploy doc for hyperion

### PR DESCRIPTION
**Pull request by Shubham9t9**
# Description

Documentation for deploying charts in hyperion

### CODE REVIEW

File: docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Chart Operations 
+
+## Introduction
+
+Discover, Create, Deploy, Update, Upgrade, Delete charts.
+
+## 1. Discover the chart from the Chart Store
+
+Select the `Charts` section from the left pane, you will be landed to the `Chart Store` page.
```
Comment (prakarsh-dt) [inline docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md:9]: you will land on the `Chart Store` page.

File: docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## 2. Configure the Chart
+
+After selecting the version and values, click on `Deploy`
+
+Enter the following details, to deploy chart:
+
+| Key | Description |
+| :--- | :--- |
+| `App Name` | Name of the Chart Unique|
+| `Project` |  Project in which you want to deploy the chart |
```
Comment (prakarsh-dt) [inline docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md:27]: Is Project required in hyperion mode too?

Comment (Shubham9t9) [reply docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md]: No. removed the project field in doc.

File: docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md (hunk 1 above)
Comment (Shubham9t9) [reply docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md]: done

File: docs/hyperion/user-guide/deploy-charts/overview-of-charts.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Select the Chart Version that you want to use and Chart Value, you can either use the Default Values or Custom Values.
+
+To know about Custom Values, Click On: [Custom Values](overview-of-charts.md#custom-values)
+
+The configuration values can be edited in the section given below Chart Version.
+
+| Key | Description |
+| :--- | :--- |
+| `App Name` | Name of the app |
+| `Project` | Project of the app |
```
Comment (prakarsh-dt) [inline docs/hyperion/user-guide/deploy-charts/overview-of-charts.md:26]: This Project?

Comment (prakarsh-dt) [reply docs/hyperion/user-guide/deploy-charts/overview-of-charts.md:26]: @Shubham9t9 ?

Comment (Shubham9t9) [reply docs/hyperion/user-guide/deploy-charts/overview-of-charts.md]: Removed project

**Pull request closed without merging**